docs/allneeds.rst

```diff
@@ -0,0 +1,16 @@
+allneeds
+========
+
+.. dfhack-tool::
+    :summary: Show the cummulitative needs of all citizens.
```

Comment:
```suggestion
    :summary: Show the cumulative needs of all citizens.
```

Reply:
Wow I really went ham with the cumulitative, need more coffee